include/attributes.h

```diff
@@ -8,9 +8,10 @@
     #endif
 #endif
 
-#define UNUSED      __attribute__((unused))
-#define FALLTHROUGH __attribute__((fallthrough))
-#define NORETURN    __attribute__((noreturn))
-#define NO_REORDER  __attribute__((no_reorder))
+#define UNUSED       __attribute__((unused))
+#define FALLTHROUGH  __attribute__((fallthrough))
+#define NORETURN     __attribute__((noreturn))
+#define NO_REORDER   __attribute__((no_reorder))
+#define SECTION_DATA __attribute__((section(".data")))
```

Comment:
I'm a little reluctant to do this now since it isn't needed anywhere else but if we ever need to use a section attribute elsewhere for a different section it could be changed in the future pretty easily (just replace `SECTION_DATA` usages in `soundfont_compiler.c` with `SECTION(.data)` or similar)